### PR TITLE
Reversing labs v2.2.0

### PR DESCRIPTION
**Pull request by MislavReversingLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Playbooks
+##### New: File Analysis - ReversingLabs TitaniumCloud
+- Check for sample analysis results on A1000. If the sample does not exist on A1000, upload sample to ReversingLabs A1000 appliance and automatically retrieve full & classification reports. Calculate final classification.
```
Comment (maimorag) [inline Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md:3]: Release notes should be in Pack Release Notes Format.[ For more information please see: ](https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.).
Also, can you explain about uploading a sample if it does not exist on A100? does the playbook uploading it or is the user needs to?
```suggestion
- Created a new playbook that checks for sample analysis results on A1000. If the sample does not exist on A1000, upload the sample to ReversingLabs A1000 appliance and automatically retrieve full & classification reports. Calculate final classification.
```

File: Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_2_0.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Playbooks
+##### New: URL Reputation - ReversingLabs TitaniumCloud
+- Get reputation data for the submitted URL.
```
Comment (maimorag) [inline Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_2_0.md:3]:
```suggestion
- Created a new playbook that gets reputation data for the submitted URL.
```

**Review by maimorag — COMMENTED** [review]
Hi @MislavReversingLabs ,
Thank you for your contribution!
Good work :)

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md (hunk 1 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md:3]: If the sample does not exist on A1000, the playbook uploads it.

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#### Playbooks
+##### New: File Analysis - ReversingLabs TitaniumCloud
+- Created a new playbook that checks for sample analysis results on the ReversingLabs A1000 malware analysis appliance. 
+- If the sample does not exist on A1000, it the uploads sample to ReversingLabs A1000 and automatically retrieves the detailed and classification reports.
```
Comment (maimorag) [inline Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md:4]: Good work, one last note

```suggestion
- Created a new playbook that checks for sample analysis results on the ReversingLabs A1000 malware analysis appliance. If the sample does not exist on A1000, the uploads sample to ReversingLabs A1000 and automatically retrieves the detailed and classification reports.
```

**Comment by efelmandar** [comment]
@MislavReversingLabs Thank you very much for your contribution and great job on the PR! I went through the review, and everything is looking good. Just one thing caught my eye - why do we need that extra task in the playbook to get the classification (task named "# 6 reversinglabs-a1000-get-classification"? 🤔 I'm not entirely sure why it's there,  since we already have the same task just before which gives the same output. Please check if it's not a duplicate and remove it if it is.

**Comment by MislavReversingLabs** [comment]
@efelmandar The first one (#8 reversinglabs-a1000-get-classification) is used to check if the sample exists on the A1000 appliance. The other get-classification command is used to actually fetch the classification result once the sample gets uploaded to A1000.
If the sample does exist, then, in theory, you could use the first get-classification's result.
If the sample doesn't exist, you need the second one because the first one returns "None". We can't know if the sample exist beforehand so this is the only command that we can use as a check.

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md (hunk 3 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md]: Done

File: Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_2_0.md (hunk 2 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_Titanium_Cloud/ReleaseNotes/2_2_0.md]: Done

File: Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md (hunk 1 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_A1000/ReleaseNotes/2_2_0.md]: Done

**Comment by efelmandar** [comment]
@MislavReversingLabs Thank you for your detailed explanation, in that cases I think it's ok to keep it like this - Nice work and thank you again for your contribution!

@maimorag PR is approved

**Comment by maimorag** [comment]
@MislavReversingLabs, please fix the validation errors.

**Comment by maimorag** [comment]
Hi @MislavReversingLabs, the code looks good!

We're ready for a demo.
Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

**Review by maimorag — APPROVED** [review]

**Pull request merged**